docker-compose.cyhy.yml

```
@@ -16,4 +23,5 @@ services:
       - --cyhy-report-dir=/cyhy_reports/
       - --cybex-scorecard-dir=/cybex_scorecard/
       - --db-creds-file=/run/secrets/database_creds.yml
+      - --csa-email-file=/run/secrets/csa_emails.yml
```

Matches my suggestion for the cli arguments.

```suggestion
      - --csa-emails-file=/run/secrets/csa_emails.yml
```